src/jira/client/axios.ts

```diff
@@ -6,18 +6,11 @@ import statsd from "../../config/statsd";
 import { getLogger } from "../../config/logger";
 import { metricHttpRequest } from "../../config/metric-names";
 import { createQueryStringHash, encodeSymmetric } from "atlassian-jwt";
-import {urlParamsMiddleware} from "../../util/axios/common-middleware";
+import { urlParamsMiddleware } from "../../util/axios/url-params";
 
 const instance = process.env.INSTANCE_NAME;
 const iss = `com.github.integration${instance ? `.${instance}` : ""}`;
 
-// TODO: type hack to fix custom implementation of URL templating vars
-declare module "axios" {
-	interface AxiosRequestConfig {
-		urlParams?: Record<string, string>;
-	}
-}
-
```

Comment:
Was there a reason to change it now?

Reply:
URL params were not being encoded, which is a potential security issue and good practice to do so.  It also had some very weird behavior that was creating bad usage patterns with axios.